steely/plugins/nuke.py

```diff
@@ -0,0 +1,17 @@
+'''to purge the naughtiness just incase'''
+
+
+import shutil
+
+
+__author__ = 'izaakf'
+COMMAND = '.nuke'
+
+
+PROBABLY_ENOUGH_TO_BAN_STEELY = 5000
```

Comment:
are you sure this is enough?

Reply:
it may be. If the pr is merged we can properly test, as the local steely instance doesn't get banned by my computer